src/components/menu/portal.test.tsx

```diff
@@ -46,9 +53,11 @@ describe('Portal Component', () => {
     const onClose = vi.fn();
```

Comment:
what for we have this one in this test case?
seems like it's better to have one test case with "open" and "close". And, maybe, you want to test what URLs we have in buttons? :)

Reply:
have you missed this one?